src/Containers/ActionsField/ActionsField.tsx

```diff
@@ -17,33 +17,45 @@ const ActionsField:React.FC<Props> = ({ formState, setAction }) => {
   const handleChange = (e:React.MouseEvent<HTMLElement>):void => {
     const currentTarget = e.target as HTMLElement;
 
-    let newAction = actionsCollection.filter((action:Action) => { //eslint-disable-line
-        if (action.action === currentTarget.innerText) {
-          return action
-        }
+    let newAction:(Action | undefined) = actionsCollection.find((action:Action) => {
+      if (action.action === currentTarget.innerText) {
+        return action;
+      }
+      return undefined;
     });
 
-    setSelectedAction(newAction[0]);
+    if (!newAction) {
+      newAction = {id: 1, action: 'Create an app'};
+    };
+
+    setSelectedAction(newAction);
     setIsClicked(false);
   }
 
   const toggleBlock = (): void => setIsClicked(!isClicked);
 
   const updateAction = () => {
-    let newChosenAction = actionsCollection.filter((action:Action) => { //eslint-disable-line
+    let newChosenAction: (Action | undefined) = actionsCollection.find((action:Action) => {
       if (action.action === selectedAction.action) {
-        return action
+        return action;
       }
-    })
-    setAction({...formState, action: newChosenAction[0]})
+      return undefined;
+    });
```

Comment:
Why do you use this style of writing of find function? This method allows using short version of it:
`actionsCollection.find((action:Action) => action.action === selectedAction.action)`